internal/task/task/bs/format.go

```diff
@@ -247,6 +249,10 @@ func NewFormatChunkfilePoolTask(curveadm *cli.CurveAdm, fc *configure.FormatConf
 	formatScriptPath := fmt.Sprintf("%s/format.sh", layout.ToolsBinDir)
 	formatCommand := fmt.Sprintf("%s %s %d %d %s %s", formatScriptPath, layout.FormatBinaryPath,
 		usagePercent, DEFAULT_CHUNKFILE_SIZE, layout.ChunkfilePoolDir, layout.ChunkfilePoolMetaPath)
+	debugmode := curveadm.MemStorage().Get(comm.DEBUG_MODE).(bool)
+	if debugmode {
+		glg.Get().SetLevel(glg.DEBG)
+	}
```

Comment:
no need to adjust the log level here. the log level should be determined by the configuration file.

Reply:
There are two ways to implement debug mode. The first is to directly modify curveadm.cfg and read the log level through the configuration file. The second method is to configure the debug mode through the command line, such as :
```shell
curveadm format --debug 
curveadm deploy --debug
```
After communicating with @Wine93 this time, I chose this solution. This Solution set the debug mode in the command line, which may conflict with the log level in curveadm.cfg, so it is necessary to modify the log level.

Reply:
> There are two ways to implement debug mode. The first is to directly modify curveadm.cfg and read the log level through the configuration file. The second method is to configure the debug mode through the command line, such as :
> 
> ```shell
> curveadm format --debug 
> curveadm deploy --debug
> ```
> 
> After communicating with @Wine93 this time, I chose this solution. This Solution set the debug mode in the command line, which may conflict with the log level in curveadm.cfg, so it is necessary to modify the log level.

yep, maybe you can provide a function for curveadm to set log level.